config/crons

```diff
@@ -0,0 +1,4 @@
+SHELL=/bin/bash
+BASH_ENV=/code/cron.env
+
+0 * * * * root python3 /code/app/manage.py fetch-build-history >> /var/log/cron.log
```

Comment:
I wouldn't write logs to a file which is a default file for an option made configurable by `cron` for writing CRON entries.